src/emoji/data.js

```diff
@@ -25,10 +25,17 @@ export const codeToEmojiMap = objectFromEntries<string, string>(
   }),
 );
 
-export const getFilteredEmojiNames = (
+export const getFilteredEmojis = (
   query: string,
   activeRealmEmojiByName: $ReadOnly<{ [string]: ImageEmojiType }>,
-): string[] => {
+): $ReadOnlyArray<{ name: string, code: string }> => {
   const names = [...unicodeEmojiNames, ...Object.keys(activeRealmEmojiByName)];
-  return Array.from(new Set([...names.filter(x => x.indexOf(query) === 0).sort()]));
+  return Array.from(new Set([...names.filter(x => x.indexOf(query) === 0).sort()])).map(
```

Comment:
Ya. 

But I thinks we should `includes ` so that, it searched letters in mid of the word as well like web. Thoughts?

```suggestion
  return Array.from(new Set([...names.filter(x => x. includes(query)).sort()])).map(
```

Reply:
Whatever the webapp does here, we should probably do the same. That's independent of what this PR is about, though. If our current behavior is different from the webapp's, want to file an issue explaining that?

Reply:
Opened https://github.com/zulip/zulip-mobile/issues/3948